commit/core.pyx

```diff
@@ -207,6 +210,64 @@ cdef class Evaluation :
         LOG( '   [ %.1f seconds ]' % ( time.time() - tic ) )
 
 
+    def load_confidence_map( self, confidence_map_filename ) :
+        """Load the confidence map.
+
+        Parameters
+        ----------
+        confidence_map_filename : string
+            The file name of the confidence map, relative to the subject folder 
+        
+        """
+
+        self.confidence_map  = None
```

Comment:
Why two global variables?

Reply:
True, or this implementation only one is needed.